### ISSUE TITLE
Add doctests to test suite

### ISSUE THREAD

**Issue by erikmannerfelt** [GlacioHack/xdem#171]
Doctests are great to show minimal examples:
```python
def add(a, b):
     """
     Simply adds two numbers

     :examples:
         >>> add(1, 2)
         4
     """
     return a + b
```
This will gracefully fail, as the example doesn't return the "expected" number:
```md
**********************************************************************
File "/home/erik/hello.py", line 6, in hello.add
Failed example:
    add(1, 2)
Expected:
    4
Got:
    3
**********************************************************************
1 items had failures:
   1 of   1 in hello.add
***Test Failed*** 1 failures.
```

There are a handful of doctests already in xdem, but these are not run when running pytest.

That should change!!

**Comment by adehecq** [comment]
How do you do that ?!?

**Comment by erikmannerfelt** [comment]
Fixed in #172
